bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/async/TestAsyncColumnFamily.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Please put `Google LLC` instead of `Google Inc.`

Reply:
Done